### DISCUSSION TITLE
GitHub Copilot for teams

### DISCUSSION THREAD

**Discussion by adobs** [community/community#19702] [Copilot Conversations] [answered]
When will GitHub Copilot be available for team licenses?  Thanks!

**Comment by D7EAD** [comment, marked as answer]
Hi!

As of right now, Github Copilot does not appear to support Enterprise-managed or institution-managed user accounts and plans. However, expect these licenses (including Team licenses) to be available in the near future!

If I've managed to help you, kindly consider **Marking as Answer** so others with the same question can more easily find the appropriate answer. Thanks!

**Reply by adobs** [reply]
@D7EAD  Is there a way to subscribe to be notified of when this will happen?  Or do I need to keep checking Github Copilot's page every few months?  Tx!

**Reply by D7EAD** [reply]
@adobs,

There is an Explore Newsletter you can subscribe to [here](https://github.com/explore/email), however, I believe this only covers the Explore page and not Github updates. There is also the Github Blog [Newsletter](https://github.blog/#newsletter) that, I believe, would serve as a good way to be notified.

**Reply by adobs** [reply]
ty!

**Comment by BartJanvanAssen** [comment]
When will this arrive?

**Comment by SyntaxError843** [comment]
Yeah like an organization subscription of like 5 people max for about 30$ a month man that'd be cool, but of course capitalism gonna make that like 45$ plus tax plus 3 of the members' nutsacks plus ratio :moyai: :moyai: :moyai:

**Comment by sweepy84** [comment]
Waiting on this to rollout to whole team. Def. not doing this one by one on each developers account.

Any updates?

**Comment by sergey-king** [comment]
subscribe

**Comment by adihoward** [comment]
Hey, my team is also waiting for this.
Can you please prioritize this and tell us exactly when it will be available?
It will actually make Gihub richer - worth prioritizing.
Thanks.

**Comment by GalDayan** [comment]
Hey, I would like to buy for my team Github Copilot (I'm the only payer) how can I do so?

**Comment by ndeeH** [comment]
Is Copilot included in [GitHub Team license](https://github.com/pricing#compare-features)?
It is excluded in Free license - but the team license says:
> ⬅ Everything included in Free, plus...

**Comment by squpshift** [comment]
@github-staff what's the hold up? Can't you just get co-pilot to write the code to enable co-pilot for teams?
